browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -32,6 +33,11 @@ namespace {
 constexpr char kSessionStorageSuffix[] = "/ephemeral-session-storage";
 constexpr char kLocalStorageSuffix[] = "/ephemeral-local-storage";
 
+const base::TimeDelta kStorageKeepAliveDelay =
+    net::features::kTimeoutTcpConnectAttemptMax.Get();
+
+base::TimeDelta g_storage_keep_alive_for_testing = base::TimeDelta::Min();
```

Comment:
for tests you can use ```base::test::ScopedFeatureList``` with an [initializer](https://github.com/chromium/chromium/blob/91.0.4450.1/base/test/scoped_feature_list.h#L108-L110) that takes ```FeatureAndParams``` values.

Reply:
I'm removing `kTimeoutTcpConnectAttemptMax` because we are going to follow-up with a PR to allow multiple options for the timeout